src/api/network-local-peering.tsx

```diff
@@ -0,0 +1,96 @@
+import { handleResponse } from "util/helpers";
+import type { LxdNetworkPeer } from "types/network";
+import type { LxdApiResponse } from "types/apiResponse";
+
+export const fetchNetworkPeers = async (
+  network: string,
+  project: string,
+): Promise<LxdNetworkPeer[]> => {
+  const params = new URLSearchParams();
+  params.set("project", project);
+  params.set("recursion", "1");
+
+  return fetch(
+    `/1.0/networks/${encodeURIComponent(network)}/peers?${params.toString()}`,
+  )
+    .then(handleResponse)
+    .then((data: LxdApiResponse<LxdNetworkPeer[]>) => {
+      return data.metadata;
+    });
+};
+
+export const fetchNetworkPeer = async (
+  network: string,
+  project: string,
+  localPeering: string,
+): Promise<LxdNetworkPeer[]> => {
```

Comment:
```suggestion
): Promise<LxdNetworkPeer> => {
```
This shouldn't be an array, right?